components/modules/Kconfig

```diff
@@ -67,6 +67,12 @@ config LUA_MODULE_BTHCI
   help
       Includes the simple BlueTooth HCI module.
 
+config LUA_MODULE_CAN
```

Comment:
ESPCAN node was from original driver, but not necessary for nodemcu. I removed components/driver_can/Kconfig.